packages/sdk-cli/src/models/HostConnections.ts

```diff
@@ -64,11 +61,11 @@ class HostConnections {
   appHost?: HostConnection;
   companionHost?: HostConnection;
 
-  async connect(hostType: HostType, hostID: string) {
+  async connect(hostType: HostType, hostWs: stream.Duplex) {
```

Comment:
nit: technically you don't care this is a WebSocket so `hostStream` might be more appropriate of a name